INSTALL.md

```diff
@@ -277,6 +297,19 @@ docker run -p 9001:9000 --name clamav -d niilo/clamav-rest
 	        name: file-uploader
 	        maxAge: 15552000000 // 1000 * 60 * 60 * 24 * 180 (6 months)
 	    }
+    	email {
```

Comment:
weird indentation, there should be a ":" here.

Reply:
fixed.